pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=61.0"]
+requires = ["setuptools>=70.0"]
```

Comment:
I believe `>=63` would be enough, but since I'm not quite sure `>=70` definitely covers it.